src/components/panels/hours-panel-date-viewer.js

```diff
@@ -0,0 +1,586 @@
+import { Heading, Icon, Link, Margins, MEDIA_QUERIES, SPACING, TYPOGRAPHY } from '../../reusable';
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { useStateValue } from '../use-state';
+
+const dateFormat = (string, abbreviated = false) => {
+  if (abbreviated) {
+    return string.toLocaleString('en-US', {
+      day: 'numeric',
+      month: 'short'
+    });
+  }
+  return string.toLocaleString('en-US', {
+    day: 'numeric',
+    month: 'long',
+    weekday: 'long',
+    year: 'numeric'
+  });
+};
```

Comment:
Not a blocker, but an opportunity to reduce code:
```suggestion
const dateFormat = (date, abbreviated = false) => {
  const options = {
    day: 'numeric',
    month: abbreviated ? 'short' : 'long',
    ...(abbreviated ? {} : { weekday: 'long', year: 'numeric' })
  };
  
  return date.toLocaleString('en-US', options);
};
```

Reply:
+1 for less code! Always in favor of that. Thanks!